llvm/lib/Transforms/Vectorize/VPlanTransforms.h

```diff
@@ -58,17 +58,21 @@ struct VPlanTransforms {
                                                                 LoopInfo &LI);
 
   /// Prepare the plan for vectorization. It will introduce a dedicated
-  /// VPBasicBlock for the vector pre-header as well as a VPBasicBlock as exit
-  /// block of the main vector loop (middle.block). If a check is needed to
+  /// VPBasicBlock for the vector pre-header, a VPBasicBlock as exit
+  /// block of the main vector loop (middle.block) and a VPBaiscBlock for the
```

Comment:
```suggestion
  /// block of the main vector loop (middle.block) and a VPBasicBlock for the
```

Reply:
That should be fixed in the latest version, thanks